showcase/app/components/shw/text/index.ts

```diff
@@ -115,14 +128,14 @@ export default class ShwTextIndexComponent extends Component {
    * @return {string} The "class" attribute to apply to the component.
    */
   get classNames() {
-    let classes = ['shw-text'];
+    const classes = ['shw-text'];
 
     // Add a class based on the @variant or @tag arguments
     classes.push(`shw-text-${this.variant}`);
 
     // Add a class based on the @weight argument
     if (this.weight && this.weight !== 'inherit') {
-      classes.push(`shw-text-weight-${this.variant.weight}`);
```

Comment:
`this.variant.weight` doesn't exist and judging from how the align part works below, I'm guessing it should just be `this.weight`.

Not sure if this would change the showcase but we'll see!